documentation/modules/proc-configuring-kafka-bridge-smr-metrics.adoc

```diff
@@ -0,0 +1,35 @@
+[id='proc-configuring-kafka-bridge-smr-metrics-{context}']
```

Comment:
I don't know if the bridge has metrics dashboards but if so they probably need updating when using the metrics reporter.